### PR TITLE
add elasticsearch widgets

### PR DESCRIPTION
**Pull request by meirwah**
## Status
In Progress

## Description
elasticsearch health widgets

## Required version of Demisto
5.5.0

## Does it break backward compatibility?
   - No

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](link)
- [ ] [CHANGELOG](link)

### CODE REVIEW

File: Widgets/widget-ElasticCPUPercentage.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+                },
+                "#00CD33": {
+                    "value": -1
+                }
+            },
+            "type": "above"
+        }
+    },
+    "size": 0,
+    "description": "Elasticsearch CPU Current Usage"
```
Comment (Bars92) [inline Widgets/widget-ElasticCPUPercentage.json]:
```suggestion
    "description": "Elasticsearch CPU current usage percentage"
```

File: Widgets/widget-ElasticDiskUsageAvg.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        },
+        "#00CD33": {
+          "value": -1
+        }
+      },
+      "type": "above"
+    }
+  },
+  "size": 0,
+  "description": "Elastic Disk Current Usage"
```
Comment (Bars92) [inline Widgets/widget-ElasticDiskUsageAvg.json]:
```suggestion
  "description": "Elastic Disk current usage percentage"
```

File: Widgets/widget-ElasticMemoryUsageAvg.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        },
+        "#00CD33": {
+          "value": -1
+        }
+      },
+      "type": "above"
+    }
+  },
+  "size": 0,
+  "description": "Elastic Memory Current Usage"
```
Comment (Bars92) [inline Widgets/widget-ElasticMemoryUsageAvg.json]:
```suggestion
  "description": "Elastic Memory current usage percentage"
```

File: dev-requirements-py3.txt [hunk 4]
```diff
@@ -1,5 +1,5 @@
 flake8==3.7.8
-demisto-sdk>=0.3.2
+demisto-sdk>=0.3.3
```
Comment (Bars92) [inline dev-requirements-py3.txt]: what is that?

Comment (meirwah) [reply dev-requirements-py3.txt]: content build issue, which is fixed with new SDK

**Comment by kirbles19** [comment]
@meirwah done.

**Pull request merged**